whelk-core/src/main/groovy/whelk/converter/marc/MarcFrameConverter.groovy

```diff
@@ -2880,6 +2882,17 @@ class MarcSubFieldHandler extends ConversionPart {
                 continue
             }
 
+            // TODO: Later, it may be desirable to add functionality
+            // to specify the punctuation mark to be used when merging.
+            if (onRevertMergeWith) {
+                if (entity.containsKey(onRevertMergeWith)) {
+                    String valueToAppend = entity[onRevertMergeWith] instanceof List ?
```

Comment:
Name-nitpicking: maybe call it something like `onRevertAppendValueFrom`?

Reply:
haha, I had actually named it to `onRevertAppendProperty` first which is almost the same as your suggestion. I'll change it to your suggestion :) Naming things is not the easiest! :P